app/views/admin/rebate_forms/index.html.haml

```diff
@@ -33,12 +33,12 @@
         %td
           - if rebate_form.council.present?
             = link_to rebate_form.council.name, admin_council_path(rebate_form.council)
-        %td= rebate_form.valuation_id
+        %td= rebate_form.fields['valuation_id']
         %td= rebate_form.fields['full_name']
         %td
           - if rebate_form.property.present?
             %p
-              = rebate_form.property.location
+              = rebate_form.fields['address']
```

Comment:
Not sure why this changed. We shouldn't use the address from the fields hash. I didn't know we had one. We just use the one in the council database for the property we are claiming on.